lib/graphs/digraph.nit

```diff
@@ -0,0 +1,606 @@
+# Implementations for representing directed graphs, also called digraphs.
+#
+# Currently, there are two classes
```

Comment:
I think that `there are two classes` is not a really helpful information. Furthermore, it can be outdated if you add more classes and `nitdoc` already list those classes for you.

The interesting thing to mention here would be what class or classes should I look at first to use your library?